internal/support/notifications/init.go

```diff
@@ -187,7 +187,7 @@ func connectToConsul(conf *ConfigurationStruct) (*ConfigurationStruct, error) {
 		}
 		conf = actual
 		//Check that information was successfully read from Consul
-		if conf.ResendLimit == 0 {
```

Comment:
Likewise, I'm trying to understand how / why the need for this logical check (i.e. `conf.ResendLimit == 0`) was obviated instead of perhaps 'rolling' this logical check into (or alongside) the one for `conf.Service.Port == 0`.